internal/lsp/cache.go

```diff
@@ -30,6 +30,9 @@ type Cache struct {
 	// diagnosticsParseErrors is a map of file URI to parse errors for that file
 	diagnosticsParseErrors map[string][]Diagnostic
 	diagnosticsParseMu     sync.Mutex
+
+	builtinPositionsFile map[string]map[uint][]BuiltinPosition
```

Comment:
'File' seemed unexpected in this name to me since the index is by URI, and the contents isn't files but rather by line. idk, minor one so feel free to ignore!

Reply:
Right, but the URI points to a file, no?

Reply:
Sometimes the URI is a folder, but that doesn't apply to hover/built ins clearly.